### DISCUSSION TITLE
Have you ever considered PostGraphile as core for the DB related functionalities?

### DISCUSSION THREAD

**Discussion by vabatta** [supabase/supabase#2525] [Feature Requests]
I have been browsing the open-source community for a while, and in the journey of testing different solution for database, PostGraphile (https://www.graphile.org/postgraphile/) totally got me. I was wondering if this project ever came across it, and thought of it as a viable solution instead of relying upon PostgRest (and Realtime).

**Comment by dqii** [comment]
https://github.com/supabase/supabase/discussions/1057
Also hoping for a PostGraphile integration in the future, but it doesn't look like it'll likely be coming soon based on the above discussion
